setup.py

```diff
@@ -31,7 +31,8 @@
     "web3==5.19.0",
     "cryptography==3.3.2",
     "scipy",
-    "enforce-typing==1.0.0.post1"
+    "enforce-typing==1.0.0.post1",
+    "json-sempai==0.4.0",
```

Comment:
Interestingly, if you implement my commments for removing unused imports from `contract_utils.py` and `utils.py`, then nothing in `ocean.py` will be using `json-sempai`.  Should we remove it from the dependency list?

Reply:
actually it will stay, since the magic I'm doing with importlib uses the json-sempai magic in turn :)